script/verdaccio.js

```diff
@@ -0,0 +1,277 @@
+  process.chdir(path.join(`./${SDK_APP_NAME}`));
+  
+  // Update package.json version
+  let package = fs.readFileSync('package.json', 'utf-8');
+
+  const packageJson = JSON.parse(package);
+  const originalVersion = packageJson.version;
+  const versions = originalVersion.split('.').map(x => parseInt(x));
+  versions[2] += 1;
+  const newVersion = versions.join('.');
+
+  logger.log(`Changing version number from ${originalVersion} to ${newVersion}`);
+
+  spawnOrFail('npm', [`version`,`${newVersion}`, '--no-git-tag-version']);
+
+  package = fs.readFileSync('package.json', 'utf-8');
+
+  logger.log(`Temporarily removing prepublishOnly and postpublish`);
+  fs.writeFileSync('package.json', package.replace(`"postpublish": "script/postpublish",`, ""));
+  package = fs.readFileSync('package.json', 'utf-8');
+  fs.writeFileSync('package.json', package.replace(`"prepublishOnly": "script/publish",`, ""));
+  
+  logger.log('Installing dependencies...');
+  spawnOrFail('npm', ['install']);
+  logger.log('Building...');
+  spawnOrFail('npm', ['run', 'build']);
+
+  try {
+    // Cleanup package before publishing new
+    logger.log('Unpublishing any existing packages from verdaccio.');
+    cp.execSync(`npm unpublish --registry ${verdaccioEndpoint} --force`);
+  } catch {
+    logger.log('No existing npm module in verdaccio.');
+  }
+  logger.log('Publishing package to verdaccio.');
+  spawnOrFail('npm', [`publish`, '--registry', `${verdaccioEndpoint}`]);
+  
+  process.chdir('./demos/browser');
+
+  spawnOrFail('npm', [`uninstall ${SDK_APP_NAME}`]);
+
+  //  Install SDK components package from local repo.
+  logger.log(`Installing ${SDK_APP_NAME} from local verdaccio server...`);
+  spawnOrFail('npm', [`install ${SDK_APP_NAME} --registry ${verdaccioEndpoint}`]);
+
+  let meetingV2File = fs.readFileSync('app/meetingV2/meetingV2.ts', 'utf-8');
+  fs.writeFileSync('app/meetingV2/meetingV2.ts', meetingV2File.replace('../../../../src/index', 'amazon-chime-sdk-js'));
+
+  logger.log(`Installing...`);
+  cp.execSync('npm install');
+  cp.execSync('npm install aws-sdk');
+  logger.log(`Building...`);
+  cp.execSync('npm run build');
+  const demoAppClientProcess = cp.exec('node server.js');
+  applicationProcesses.push(demoAppClientProcess);
+  
+  logger.log(`
+    Starting meeting demo app. Please wait for the server to start up.
+    App will be running on: http://127.0.0.1:8080/
+
+    Once application is loaded verify that its working properly.`
+  );
+  
+  logger.warn(
+    `Have you completed the demo app manual verification steps and are ready to continue with RELEASE?`
+  );
+  shouldContinuePrompt(cleanupBeforeExit);
+  demoAppClientProcess.kill();
+
+  // Start Backwards compatible testing
+  logger.log("Starting Backwards Compatibility Testing")
+  // Clean up
+  process.chdir('../..');
+  spawnOrFail('rm', ['-rf', `${SDK_APP_NAME}`]);
+
+  let previousVersion = prompt(
+    `Please type the previous NPM version to continue: `
```

Comment:
Could we just get the latest version from github (there should be a tag for that release)